.github/workflows/macos.yml

```diff
@@ -39,12 +39,12 @@ jobs:
           - host: macos-15
             arch: x86_64
             minimum_deployment: '11.0'
-            max_warnings: 61
+            max_warnings: 9
 
           - host: macos-15
             arch: arm64
             minimum_deployment: '11.0'
-            max_warnings: 61
+            max_warnings: 9
```

Comment:
Huge improvement! 💯

Reply:
I intend to look into the remaining ones someday, but definitely after your PR https://github.com/dosbox-staging/dosbox-staging/pull/4351 is merged.